odl/tomo/geometry/geometry.py

```diff
@@ -155,83 +173,189 @@ def translation(self):
         """Shift of the origin of this geometry."""
         return self.__translation
 
-    def det_refpoint(self, mpar):
+    @property
+    def check_bounds(self):
+        """Whether to check if method parameters are in the valid range."""
```

Comment:
Any suggestion on how to call these methods? I guess it would be a short version of "methods that compute vectors associated with the geometry"